src/OpenRiaServices.Hosting.AspNetCore/Framework/AspNetCore/OpenRiaServicesOptions.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using Microsoft.AspNetCore.Http;
+using OpenRiaServices.Server;
+
+#nullable enable
+
+namespace OpenRiaServices.Hosting.AspNetCore
+{
+    public sealed class OpenRiaServicesOptions
+    {
+        // TODO: Naming, should it be a single parameter, or should exception details (exception, ServiceContext) be passed as separate parameters?
+        public Action<UnhandledExceptionParameters>? UnhandledException { get; set; }
+        // public OpenRiaServicesOptions WithOnUnhandledException(Action<Exception, DomainServiceErrorInfo>) { }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether exception stack trace information should be included in error messages.
+        /// <para>This is considered INSECURE since it gives an attacker to much information</para>
+        /// </summary>
+        public bool IncludeExceptionStackTraceInErrors { get; set; }
+
+        /// <summary>
+        /// UNSAFE: Gets or sets a value indicating whether exception message details (from exceptions other than <see cref="DomainException"/>) should be included in error messages.
+        /// <para>This is generally considered INSECURE since it can provide an attacker with to much information</para>
+        /// </summary>
+        public bool IncludeExceptionMessageInErrors { get; set; }
+
+        /* ************ SOME POSSIBLE FUTURE OPTIONS ************ 
+         * 
+         * int MaxReceiveSize / MaxRequestSize { get; set; }
+         * int MaxResponseSize { get; set; }
+         * 
+         * int RouteOrder { get; set; }
+         * */
+    }
+
+    // TODO: 1: Naming, 2: Move to separate file
+    /// <summary>
+    /// <see cref="UnhandledExceptionParameters"/> can be used to control how errors are passed to the client
+    /// </summary>
+    /// <param name="exception"></param>
+    /// <param name="fault"></param>
+    public sealed class UnhandledExceptionParameters(Exception exception, DomainServiceFault fault, DomainService domainService, Microsoft.AspNetCore.Http.HttpContext httpContext)
+    {
+        public Exception Exception { get; } = exception;
+        //public DomainService? DomainService { get; } = domainService; // om enkelt att komma åt ananrs ServiceContext
+        public DomainServiceContext ServiceContext { get; } = domainService.ServiceContext;
+        public HttpContext HttpContext { get; } = httpContext;
```

Comment:
Exposing HttpContext  gives access to httpHeaders etc, but this would be the only place where we do so.
An end user can always get a `IHttpContextAccessor` using ServiceContext so maybe we should just skip the parameter

```suggestion
        internal HttpContext HttpContext { get; } = httpContext;
```